components/Navigation/NavigationContainer/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { FormattedMessage } from 'react-intl';
+import classnames from 'classnames';
+import styles from './index.module.scss';
+import type { FC, ReactNode } from 'react';
+
+type Props = {
```

Comment:
Although it's not exported but let's change it to `NavigationContainerProps` for consistency with other components. 🚀

Reply:
FYI, we have these for conventions:
- https://github.com/nodejs/nodejs.org/blob/main/COLLABORATOR_GUIDE.md#best-practices-when-creating-a-component
- https://github.com/nodejs/nodejs.org/blob/main/TRANSLATION.md#adding-new-translation-keys -> about the translation key

I also don't remember all of the conventions but I always refer to those 2 for guidance when migrating. 😄